README.md

```diff
@@ -22,7 +22,7 @@ For more details, see our [FAQ](https://www.cockroachlabs.com/docs/frequently-as
 
 ## Status
 
-CockroachDB is currently in alpha. See our
+CockroachDB is currently in beta. See our
```

Comment:
"CockroachDB is currently in beta" - sounds so official... waiting for the merge on this PR!